client/js/actions/assessment_progress.js

```diff
@@ -52,12 +52,14 @@ export const submitAssessment = (
     identifier, assessmentId, questions, answers, settings, outcomes
   });
 
-export const nextQuestion = () => ({
-  type: Constants.ASSESSMENT_NEXT_QUESTION
+export const nextQuestions = (pageSize = 1) => ({
```

Comment:
Added paging parameter to advance currentQuestionIndex by any number of questions instead of just one